Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN \
 
 # Copy the wrapper script into the container
 COPY bootstrap-script.sh /usr/local/bin/bootstrap-script.sh
+COPY hybrid-ssm-setup.sh /usr/local/bin/hybrid-ssm-setup.sh
+COPY hybrid-iam-ra-setup.sh /usr/local/bin/hybrid-iam-ra-setup.sh
```

Comment:
For binaries that are invoked as CLI commands, I prefer to omit the `.sh` extension, because it's generally not relevant what language they're written in, only what the interface is.

This is inconsistent with `bootstrap-script.sh` though that's OK; it should probably be renamed also, but you don't have to worry about that.